### DISCUSSION TITLE
Question about Backup feature

### DISCUSSION THREAD

**Discussion by FoksVHox** [pterodactyl/panel#3063] [Support] [answered]
Hello!
I wonder, as the documentation only shows an example of AWS S3 backup configuration, are there other Laravel Filesystem drivers that are supported by Pterodactyl?

**Comment by matthewpi** [comment, marked as answer]
Backups don't use the Laravel Filesystem for storage, wings handles all the uploading and processing.  Similar packages used by Laravel's Filesystem are used for S3 due to the way we handle authentication (through the use of pre-signed URLs rather than giving wings the actual credentials).  This means if you wanted to for example add SFTP support; you would need to do it in wings in order to handle the file uploading, and potentially add endpoints to the panel to allow wings to get the credentials/access for it.

**Reply by FoksVHox** [reply]
Essentially what you're telling me is that local backups aren't possible with the back system at its current stage?

**Reply by matthewpi** [reply]
Local backups are possible and are the default setup shipped with wings and the panel.  Any other external provider or integration will need to be implemented in wings manually.
